Assets/Scripts/HapticsController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HapticsController : MonoBehaviour
+{
+    private static HapticsController _hapticsController;
+    //singleton instance
+    public static HapticsController Instance
+    {
+        get
+        {
+            if (!_hapticsController) _hapticsController = FindObjectOfType(typeof(HapticsController)) as HapticsController;
+            return _hapticsController;
```

Comment:
Should send a `Debug.LogError` if _hapticsController is still null because that means there isn't an active HapticsController script in the scene